ci/prow/Makefile

```diff
@@ -19,27 +19,36 @@ JOB_NAMESPACE ?= test-pods
 
 PROW_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 TESTGRID_DIR := $(PROW_DIR)/../testgrid
+UNSET_CONTEXT := kubectl config unset current-context
 
 get-cluster-credentials:
 	gcloud container clusters get-credentials "$(CLUSTER)" --project="$(PROJECT)" --zone="$(ZONE)"
 
+unset-cluster-credentials:
```

Comment:
nit, would be nice to reuse this target instead of `UNSET_CONTEXT`

Reply:
I find `make unset-cluster-credentials` output too verbose and hard to read, thus the trick.